### PR TITLE
Created kijijiCar endpoint for card view

### PR DESCRIPTION
**Pull request by nhistory**
AC
- /api/kijijiCar endpoint

### CODE REVIEW

File: app/models/KijijiCar.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Pydantic allows auto creation of JSON Schemas from models
```
Comment (aurora-yn) [inline app/models/KijijiCar.py:1]: Please update the file with lower case and no camel case.

File: app/models/KijijiCar.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+from pydantic import BaseModel
+
+
+class KijijiCar(BaseModel):
+  car_info: str
+  url: str
+  mileage: str
+  region: str
+  price: str
```
Comment (aurora-yn) [inline app/models/KijijiCar.py:10]: For `mileage` and `price` in `usedCar`, it is int. Any reasons why this model should be str?

File: app/services/database.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # import model
```
Comment (aurora-yn) [inline app/services/database.py:1]: Left a separate comment for this file in our channel

**Review by aurora-yn — CHANGES_REQUESTED** [review]

File: app/models/KijijiCar.py (hunk 2 above)
Comment (jhchoi0830) [reply app/models/KijijiCar.py:10]: Don't we need to show the unit?

**Pull request closed without merging**